src/implementations/twig/components/pagination/pagination.html.twig

```diff
@@ -50,12 +50,14 @@
           {% if _item.aria_label is defined and _item.aria_label is not empty %}
             {% set _link = _link|merge({ aria_label: _item.aria_label }) %}
           {% endif %}
+          {% set _icon = {} %}
           {% if _item.link.icon is defined and _item.link.icon is not empty %}
             {% set _icon = _icon_path is not empty ? _item.link.icon|merge({ path: _icon_path }) : _item.link.icon %}
             {% set _link = _link|merge({ 'icon': _icon }) %}
           {% endif %}
           {% include '@ecl/link/link.html.twig' with _item.link|merge({
```

Comment:
didn't we say to just pass the local variables where we already merged the item.link objects? just like this:
```
{% include '@ecl/link/link.html.twig' with {
  link: _link|default({}),
  icon: _icon|default({}),
  extra_classes: "ecl-pagination__link",
} only %}
```

and yeah, doing like this we don't need to initialise the `_icon` variable as an empty object